pareto/utilities/earthquake_distance.py

```diff
@@ -0,0 +1,179 @@
+#####################################################################################################
+# PARETO was produced under the DOE Produced Water Application for Beneficial Reuse Environmental
+# Impact and Treatment Optimization (PARETO), and is copyright (c) 2021-2024 by the software owners:
+# The Regents of the University of California, through Lawrence Berkeley National Laboratory, et al.
+# All rights reserved.
+#
+# NOTICE. This Software was developed under funding from the U.S. Department of Energy and the U.S.
+# Government consequently retains certain rights. As such, the U.S. Government has been granted for
+# itself and others acting on its behalf a paid-up, nonexclusive, irrevocable, worldwide license in
+# the Software to reproduce, distribute copies to the public, prepare derivative works, and perform
+# publicly and display publicly, and to permit others to do so.
+#####################################################################################################
+
+import urllib.request
+import urllib.error
+import json
+import datetime
+import pyproj
+
+geod = pyproj.Geod(ellps="WGS84")
+mi_per_m = 0.000621371  # mi/m
+max_radius_mi = 5.59
+min_magnitude = 3
```

Comment:
Instead of `max_radius_mi` and `min_magnitude` being fixed constants, can these be arguments that are passed to the function(s)? It makes sense to leave the default values as 5.59 and 3, respectively, but I think the user should be able to change these parameters if they wish.